### ISSUE TITLE
Cannot load solution due to non-supported character

### ISSUE THREAD

**Issue by dkowollik** [pyansys/pydpf-post#28]
Hi Ramdane,
 
as requested I create this issue. You already have the rst file. It seems like that I cannot load the solution due to a non-supported character in the result info. 
```
from ansys.dpf import post
solution = post.load_solution('file.rst')

[libprotobuf ERROR C:\.conan\897de8\1\protobuf\src\google\protobuf\wire_format_lite.cc:578] String field 'ansys.api.dpf.result_info.v0.ResultInfoResponse.user_name' contains invalid UTF-8 data when serializing a protocol buffer. Use the 'bytes' type if you intend to send raw bytes. 

[libprotobuf ERROR T:\src\github\protobuf\protobuf\src\google\protobuf\wire_format_lite.cc:577] String field 'ansys.api.dpf.result_info.v0.ResultInfoResponse.user_name' contains invalid UTF-8 data when parsing a protocol buffer. Use the 'bytes' type if you intend to send raw bytes. 
Traceback (most recent call last):
  File "C:\Work\Blade\Tools\Ansys\DPF_Post\issue_read_model.py", line 2, in <module>
    solution = post.load_solution(r'E:\Alpha\Alpha_Carbon_Z16_files\dp0\SYS-1\MECH\file.rst')
  File "C:\Anaconda3\envs\PyAnsys39\lib\site-packages\ansys\dpf\post\post_utility.py", line 29, in load_solution
    _model = Model(data_sources)
  File "C:\Anaconda3\envs\PyAnsys39\lib\site-packages\ansys\dpf\core\model.py", line 52, in __init__
    self._metadata = Metadata(data_sources, self._server)
  File "C:\Anaconda3\envs\PyAnsys39\lib\site-packages\ansys\dpf\core\model.py", line 229, in __init__
    self._cache_result_info()
  File "C:\Anaconda3\envs\PyAnsys39\lib\site-packages\ansys\dpf\core\model.py", line 233, in _cache_result_info
    self.result_info = self._load_result_info()
  File "C:\Anaconda3\envs\PyAnsys39\lib\site-packages\ansys\dpf\core\model.py", line 351, in _load_result_info
    result_info = op.get_output(0, types.result_info)
  File "C:\Anaconda3\envs\PyAnsys39\lib\site-packages\ansys\dpf\core\errors.py", line 103, in wrapper
    raise DPFServerException(details) from None            
ansys.dpf.core.errors.DPFServerException: Exception deserializing response!
```
Best Regards,
Daniel

**Comment by dodu94** [comment]
Hi, I encountered a similar issue with a simple static analysis
[file.zip](https://github.com/pyansys/pydpf-post/files/7594648/file.zip)
s

**Comment by dodu94** [comment]
May it be an issue of compatibility with ANSYS 2021 R2?

I have no problems trying to read older relesases .rst file.

Also it seems that if you load  it with `ansys.dp.core.Model` it only gives a warning:
![image](https://user-images.githubusercontent.com/25747626/144249059-8298f697-bafa-42b4-99d6-fa0dbc2609ba.png)


I also saw that a workaround was proposed on the dpf.core github but if I undestood it correctly it was not definetevly fixed:
https://github.com/pyansys/pydpf-core/issues/114

**Comment by anslpa** [comment]
Hi, 
This issue has been solved in both pydpf-core and pydpf-post modules. Access to the Solution/Model should not be an issue anymore. A warning can still pop up: related documentations information had been merged in pyansys/pydpf-core#139 and #55.
